wazo_confd/plugins/user/resource.py

```diff
@@ -77,5 +77,5 @@ def put(self, id):
     @required_acl('confd.users.{id}.delete')
     def delete(self, id):
         tenant_uuids = self._build_tenant_list({'recurse': True})
-        self._middleware.delete(id, tenant_uuids)
+        self._middleware.delete(id, tenant_uuids, recursive='recursive' in request.args)
```

Comment:
nah, make it work with `recursive=true` only. Otherwise someone could use `recursive=false` to be "safe" and get screwed up

Reply:
Done. Can you check plz if what I did is what you want ? :) @pc-m